heroku/cli
# Feat: Adds fir pilot language to match the dashboard when create a fir space

## To test

1. checkout build etc.
2. Create a fir space. `heroku space:create --generation fir ...other flags and args` 
3. Observe the fir pilot language is output to the terminal
